test/verify/check-superuser

```diff
@@ -104,17 +104,18 @@ class TestSuperuser(MachineCase):
         b.check_superuser_indicator("Limited access")
 
         # Configure the sudo PAM stack to make two prompts
+        arch = m.execute("uname -m").strip()
         if "debian" in m.image or "ubuntu" in m.image:
-            self.write_file("/etc/pam.d/sudo", """
+            self.write_file("/etc/pam.d/sudo", f"""
 auth required pam_unix.so
-auth required /usr/lib/cockpit-test-assets/mock-pam-conv-mod.so
+auth required /usr/lib/cockpit-test-assets/mock-pam-conv-mod.{arch}.so
 @include common-account
 @include common-session-noninteractive
 """)
         else:
             self.write_file("/etc/pam.d/sudo", """
```

Comment:
`f"""`?

Reply:
Ach, ja klar :grin:  So much for "throw this against the wall^W^WCI and see what sticks. Fixed! (The other place, too)